gubernator/config.yaml

```diff
@@ -0,0 +1,30 @@
+---
+default_org: kubernetes
+default_repo: kubernetes
+jobs:
+  kubernetes-jenkins/logs/:
+  - ci-kubernetes-e2e-gce-etcd3
+  - ci-kubernetes-e2e-gci-gce
+  - ci-kubernetes-e2e-gci-gce-slow
+  - ci-kubernetes-e2e-gci-gke
+  - ci-kubernetes-e2e-gci-gke-slow
+  - ci-kubernetes-kubemark-500-gce
+  - ci-kubernetes-node-kubelet
+  - ci-kubernetes-test-go
+  - ci-kubernetes-verify-master
+  - kubernetes-build
+  - kubernetes-e2e-kops-aws
+  kubernetes-jenkins/pr-logs/directory/:
+  - pull-kubernetes-bazel-build
+  - pull-kubernetes-bazel-test
+  - pull-kubernetes-e2e-gce-bazel
+  - pull-kubernetes-e2e-gce-etcd3
+  - pull-kubernetes-e2e-gce-gpu
+  - pull-kubernetes-e2e-kops-aws
+  - pull-kubernetes-federation-e2e-gce
+  - pull-kubernetes-kubemark-e2e-gce
+  - pull-kubernetes-node-e2e
+  - pull-kubernetes-unit
+  - pull-kubernetes-verify
+prow_url: prow.k8s.io
+pull_prefix: kubernetes-jenkins/pr-logs/pull
```

Comment:
Personalization for header color / logo / title?

Reply:
Yes, we definitely need branding to be customisable.

Reply:
I was hoping for this to be the first PR in many ... will bubble up all the other stuff  we've had to do to Gubernator to get it working for us.

Reply:
follow-up sgtm